doc/examples/features_detection/plot_scattering_classification.py

```diff
@@ -0,0 +1,259 @@
+            |num images training set  | % error |
+            +-------------------------+---------+
+            |                    300  |  5.92   |
+            +-------------------------+---------+
+            |                  1000   |  2.7    |
+            +-------------------------+---------+
+            |                  2000   |  2      |
+            +-------------------------+---------+
+            |                  5000   |  1.32   |
+            +-------------------------+---------+
+            |                 10000   |  0.96   |
+            +-------------------------+---------+
+            |                 20000   |  0.82   |
+            +-------------------------+---------+
+            |                 40000   |  0.56   |
+            +-------------------------+---------+
+            |                 60000   |  0.51   |
+            +-------------------------+---------+
+
+- **CIFAR10 database**:
+ This is a very challenging database with
+ 10 classes and 32x32 color images. The code presented here can
+ reproduce the results obtained in Table 1 (Trans., order 1)
+ of [2]_, more specifically it can obtain 73.6% success for m=1
+ and 40000 images in the training set.
+
+**Warning**: For integration reasons, the code uses very few images
+in the datasets. To obtain full performance, you need
+to change the variable num_images to 10000 at least.
+
+.. [1] Bruna, J., Mallat, S. 'Invariant Scattering Convolutional Networks'.
+IEEE TPAMI, 2012.
+
+.. [2] Oyallon, E. et Mallat, S. 'Deep Roto-translation Scattering
+ for Object Classification'. CVPR 2015
+
+.. [3] https://en.wikipedia.org/wiki/Support_vector_machine
+"""
+#######################
+# MNIST database
+# Modules, MNIST database and functions for scattering feature generation
+import time as time
+from keras.datasets import mnist
+import numpy as np
+from skimage.filters.filter_bank import multiresolution_filter_bank_morlet2d
+from skimage.feature.scattering import scattering
+from sklearn.svm import SVC
+from sklearn.pipeline import make_pipeline
+from sklearn.metrics import accuracy_score
+from sklearn.preprocessing import MinMaxScaler, Normalizer, StandardScaler
+
+
+def load_images_mnist(px=32, num_images=500000):
+    (X_train_sm, y_train), (X_test_sm, y_test) = mnist.load_data()
+    X_train_sm = X_train_sm[0:num_images, :, :]
+    y_train = y_train[0:num_images]
+    X_test_sm = X_test_sm[0:num_images, :, :]
+    y_test = y_train[0:num_images]
+    num_images_ta = X_train_sm.shape[0]
+    num_images_te = X_test_sm.shape[0]
+    # set size to a power of 2 (32) with a zero pad border
+    X_train = np.zeros((num_images_ta, px, px))
+    X_train[:, 3:31, 3:31] = X_train_sm
+    X_test = np.zeros((num_images_te, px, px))
+    X_test[:, 3:31, 3:31] = X_test_sm
+    X_train = X_train.astype('float32')
+    X_test = X_test.astype('float32')
+    return X_train, y_train, X_test, y_test
+
+
+def load_scattering(X_train, X_test, px=32, J=3, L=8, m=2,
```

Comment:
Why do you need `px` in the arguments of this function? You have it in the shape of `X_train`